pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1402,10 +1402,40 @@ func doActivateTail(ctx *domainContext, status *types.DomainStatus,
 	if err != nil {
 		log.Errorf("setupVlans failed: %v", err)
 	}
+	setupVf(ctx.assignableAdapters, status.IoAdapterList)
 	log.Functionf("doActivateTail(%v) done for %s",
 		status.UUIDandVersion, status.DisplayName)
 }
 
+func setupVf(aa *types.AssignableAdapters, ioAdapterList []types.IoAdapter) {
+	if !aa.Initialized {
```

Comment:
Is there a reason we can only do this once at boot? If someone adds a VF could we set that up post-boot?

Reply:
If we add new VF, we will have to change device model, therefore reboot

Reply:
OK